gherkin/targeting.feature

```diff
@@ -0,0 +1,146 @@
+@rpc @in-process @targeting
+Feature: flagd json evaluation
```

Comment:
I think we should update this name to match the file name better, or vice-versa.